fhir-bulkdata-webapp/src/main/java/com/ibm/fhir/bulkdata/jbatch/export/fast/ResourcePayloadReader.java

```diff
@@ -481,9 +485,7 @@ private void uploadWhenReady() throws Exception {
     private void uploadPartToAzure() throws Exception {
         // Azure API: Part number must be an integer between 1 and 10000
         int currentObjectPartNumber = uploadedParts.size() + 1;
-        if (logger.isLoggable(Level.FINE)) {
-            logger.fine(logPrefix() + " Uploading part# " + currentObjectPartNumber + " ["+ ioBuffer.size() + " bytes] for uploadId '" + uploadId + "'");
-        }
+        logger.fine(() -> logPrefix() + " Uploading part# " + currentObjectPartNumber + " ["+ ioBuffer.size() + " bytes] for uploadId '" + uploadId + "'");
```

Comment:
uploadId will always be "azure" here right?
```suggestion
        logger.fine(() -> logPrefix() + " Uploading part# " + currentObjectPartNumber + " ["+ ioBuffer.size() + " bytes]");
```

Reply:
I guess it doesn't hurt to say "for uploadId 'azure'" though, so feel free to ignore

Reply:
Right - always azure.